From dd6c5b70fa0dc66b569a1aacd84b45e417af0372 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hannes=20M=C3=BCller?= <h.c.f.mueller@gmx.de>
Date: Wed, 26 Apr 2017 10:41:47 +0200
Subject: [PATCH] Remove version in file name of dynamic library guile-readline
To: guile-devel@gnu.org

* guile-readline/Makefile.am: Add -avoid-version to guile_readline_la_LDFLAGS.
Dynamic library guile-readline resides in a "major-version"."minor-version"
directory. Therefore no additional versioning is required. The patch allows
standard installation on MSYS2, which is done without .la files.
---
 guile-readline/Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/guile-readline/Makefile.am b/guile-readline/Makefile.am
index ade7dd09d..36c2f5415 100644
--- a/guile-readline/Makefile.am
+++ b/guile-readline/Makefile.am
@@ -51,7 +51,8 @@ guile_readline_la_LIBADD =				\
   $(READLINE_LIBS)					\
   ../libguile/libguile-@GUILE_EFFECTIVE_VERSION@.la ../lib/libgnu.la
 
-guile_readline_la_LDFLAGS = -export-dynamic -no-undefined -module
+guile_readline_la_LDFLAGS = -export-dynamic		\
+			    -no-undefined -module -avoid-version
 
 BUILT_SOURCES = readline.x
 
-- 
2.12.1

